src/condor_starter.V6.1/jic_shadow.cpp

```diff
@@ -981,6 +981,18 @@ JICShadow::notifyExecutionExit( void ) {
 	}
 }
 
+
+bool
+JICShadow::genericRequestGuidance( const ClassAd & request, GuidanceResult & rv, ClassAd & guidance ) {
+	if( shadow_version && shadow_version->built_since_version(24, 3, 0) ) {
```

Comment:
Does this version have to bumped up or is this `v24.3.0` still accurate?

Reply:
It should be bumped; thanks for the reminder.